ui/pages/settings/security-tab/reveal-srp-list/reveal-srp-list.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import { Box } from '../../../../components/component-library';
+import SRPQuiz from '../../../../components/app/srp-quiz-modal/SRPQuiz';
+import { SRPList } from '../../../../components/multichain/multi-srp/srp-list/srp-list';
+
+export const RevealSRPList = () => {
+  const [srpQuizModalVisible, setSrpQuizModalVisible] = useState(false);
+  const [selectedKeyringId, setSelectedKeyringId] = useState('');
+
+  return (
+    <Box>
+      <SRPList
+        onActionComplete={(keyringId) => {
+          setSelectedKeyringId(keyringId);
+          setSrpQuizModalVisible(true);
+        }}
+        hideShowAccounts={true}
+      />
+      {srpQuizModalVisible && selectedKeyringId && (
+        <SRPQuiz
+          keyringId={selectedKeyringId}
+          isOpen={srpQuizModalVisible}
+          onClose={() => setSrpQuizModalVisible(false)}
+        />
+      )}
```

Comment:
Should probably be named `SRPQuizModal` since this one is displayed "on-top" of the `SRPList` component (I was confused when reading the code at first)